manager.go

```diff
@@ -0,0 +1,248 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the LGPL, see LICENCE file for details.
+
+package aclstore
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/julienschmidt/httprouter"
+	"gopkg.in/errgo.v1"
+	httprequest "gopkg.in/httprequest.v1"
+
+	"github.com/juju/aclstore/params"
+)
+
+// Params holds the parameters for a NewManager call.
+type Params struct {
+	// Store holds the persistent storage used by the handler.
+	Store ACLStore
+
+	// RootPath holds the root URL path prefix to use
+	// for the ACL endpoints. All the endpoints will be
+	// prefixed with this path.
+	RootPath string
+
+	// Authenticate authenticates the given HTTP request and returns
```

Comment:
I don't think this comment is clear enough. It makes it look like the error is optional if authentication fails. I assume that you do actually have to return an error on failure (if not please make that explicit too).

Reply:
done